src/storage/src/hummock/observer_manager.rs

```diff
@@ -20,6 +20,8 @@ use risingwave_common_service::observer_manager::{ObserverState, SubscribeHummoc
 use risingwave_hummock_sdk::filter_key_extractor::{
     FilterKeyExtractorImpl, FilterKeyExtractorManagerRef,
 };
+#[cfg(hm_trace)]
```

Comment:
We should limit the appearance of #[cfg(hm_trace)] to `monitor/mod.rs`. 

We can do `use crate::monitor::trace` and call `trace!` anyway. And in monitor/mod.rs, when `hm_trace` is enabled, we do `pub use risingwave_hummock_trace::trace`, and otherwise implement a macro named trace that does nothing.